app/views/users/_account.html.erb

```diff
@@ -17,7 +17,7 @@
 <% end %>
 
 <h2>DEV API Access Tokens</h2>
-<p>You can generate personal access tokens to use for authentication with the DEV API. The API is still in early beta though.</p>
+<p>You can generate personal access tokens to use for authentication with the DEV API. The API is still in early beta though. The DEV API documentation can be found at <a href="/api/" ></a></p>
```

Comment:
I think you're missing the text inside the link.

I'd say: 

```html
The <a href="/api">DEV API documentation</a> contains further information.
```

Even if right now contains only info on the Articles create/update endpoints, eventually it will contain all the API doc.

Reply:
Fixed. Sorry about that 🤦🏻‍♂️